data-model/fdo-types/0.3.0/digital-specimens/schema/material-entity.json

```diff
@@ -0,0 +1,162 @@
+{
+  "$id": "https://schemas.dissco.tech/schemas/digitalobjects/0.3.0/digital-specimens/material-entity.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "$comment": "DigitalObject Version 0.3.0, only if the part did not get a separate catalogue number, otherwise it will be a separate digital specimen itself",
+  "properties": {
+    "@id": {
+      "type": "string",
+      "description": "The identifier for the digital object."
+    },
+    "@type": {
+      "type": "string",
+      "const": "ods:MaterialEntity",
+      "description": "The type of the digital object, in this case a ods:MaterialEntity"
+    },
+    "dwc:materialEntityID": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/materialEntityID",
+      "examples": [
+        "06809dc5-f143-459a-be1a-6f03e63fc083"
+      ]
+    },
+    "ods:materialEntityType": {
+      "type": "string",
+      "description": "Type of material entity",
+      "examples": [
+        "organism"
+      ]
+    },
+    "dwc:preparations": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/preparations",
+      "examples": [
+        "fossil"
+      ]
+    },
+    "dwc:disposition": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/disposition",
+      "examples": [
+        "in collection"
+      ]
+    },
+    "dwc:institutionCode": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/institutionCode",
+      "examples": [
+        "MNF"
+      ]
+    },
+    "dwc:institutionID": {
+      "type": "string",
+      "format": "(https:\/\/ror\\.org\/0[\\w]{6}[0-9]{2})|(http:\/\/www\\.wikidata.org\/entity\/\\w+)",
+      "description": "ROR or Wikidata identifier, based on https://rs.tdwg.org/dwc/terms/institutionID",
+      "examples": [
+        "https://ror.org/015hz7p22"
+      ]
+    },
+    "ods:institutionName": {
+      "type": "string",
+      "description": "Full museum name according to ROR or Wikidata",
+      "examples": [
+        "National Museum of Natural History"
+      ]
+    },
+    "dwc:collectionCode": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/collectionCode",
+      "examples": [
+        "EBIRD"
+      ]
+    },
+    "dwc:collectionID": {
+      "type": "string",
+      "description": "https://rs.tdwg.org/dwc/terms/collectionID",
+      "examples": [
+        "https://www.gbif.org/grscicoll/collection/fbd3ed74-5a21-4e01-b86a-33d36f032d9c"
+      ]
+    },
+    "dwc:ownerInstitutionId": {
```

Comment:
regex?

Reply:
ownerInstituion***ID***

Reply:
or ownerInstitutionCode? I can't find this term in dwc

http://rs.tdwg.org/dwc/terms/ownerInstitutionCode

Reply:
I think we need both dwc_ownerInstitutionCode (like "RMNH") ownerInstituionId (like ror or wikidata). but there's no dwc for ownerInstituionId. if this is covered by dwc:institutionID we can skip ownerInstituionId.